packages/aio-batch-update/README.md

```diff
@@ -0,0 +1,40 @@
+# aio-batch-update 
+
+Make batch changes across multiple repos using a Git patch
+
+## Getting Started
+
+### Pre-requisites 
+
+- [git-xargs](https://github.com/gruntwork-io/git-xargs?tab=readme-ov-file#getting-started)
+- Create classic github pat with `repo`, `workflow`, `gist`, and `user` scopes, export to local environment 
+    ```
+    export GITHUB_OAUTH_TOKEN=gph...
+    ```
+
+### Install Dependencies
+
+`npm install` 
+
+## Usage 
+
+### Generate new Git patch 
+
+- Make changes to a single repo locally
+- Before committing, run `git diff > [some-name].patch`
+- Add the new patch file to this repo for historical purposes
+
+### Fill out batch update configuration
+
+- Copy `update-config.example.json` to `update-config.json` 
+- Fill out new config file
+    - `branch` - name of branch to create
+    - `commitMessage` - message to include in the commit
+    - `repos` (optional) - list of github repos to apply patch to (if empty uses all active and non-documentation aio-app-builder-repo)
+    - `patch` - name of the Git patch in `patches` folder to apply
+
+### Run
+
+⚠️ Use with care: If `repos` is empty, this will create a branch and open a pull request across 70+ repos
```

Comment:
perhaps we need a confirm step for this case?

Reply:
totally - there's a quick inquirer confirm/deny prompt that shows how many repos the patch will be applied to here https://github.com/adobe/aio-pkg-utils/pull/18/files#diff-78c21a0c918fd057ec934c7f7dd0810c9d6b3e669d2064c26c61b5b372a8e81dR61